prdoc/pr_6039.prdoc

```diff
@@ -0,0 +1,41 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Added Trusted Query API calls."
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Added is_trusted_reserve and is_trusted_teleporter API calls to all the runtimes.
```

Comment:
```suggestion
  - audience: Runtime Dev
    description: |
      Added is_trusted_reserve and is_trusted_teleporter API calls to all the runtimes.
      Given an asset and a location, they return if the chain trusts that location as a reserve or teleporter for that asset respectively.
      You can implement them on your runtime by just calling a helper function on `pallet-xcm`.
      ```rust
      	impl xcm_runtime_apis::trust_query::TrustedQueryApi<Block> for Runtime {
		fn is_trusted_reserve(asset: VersionedAsset, location: VersionedLocation) -> Result<bool, xcm_runtime_apis::trust_query::Error> {
			PolkadotXcm::is_trusted_reserve(asset, location)
		}
		fn is_trusted_teleporter(asset: VersionedAsset, location: VersionedLocation) -> Result<bool, xcm_runtime_apis::trust_query::Error> {
			PolkadotXcm::is_trusted_teleporter(asset, location)
		}
	}
      ```
```